src/Tizen.Multimedia.MediaPlayer/Player/Player.cs

```diff
@@ -884,7 +884,54 @@ protected void ClearPreparing()
         {
             Interlocked.Exchange(ref _isPreparing, 0);
         }
-
         #endregion
+
+        /// <summary>
+        /// Decode an audio frame to export PCM from a data.
+        /// </summary>
+        /// <remarks><para>This function must be called before calling player_prepare() or player_prepare_async().
+        /// A registered callback is called in a separate thread(not in the main loop).
+        /// The audio PCM data can be retrieved using a registered callback as a media packet
+        /// and it is available until it's destroyed by media_packet_destroy().</para>
+        /// <para>The packet has to be destroyed as quickly as possible after rendering the data
+        /// and all the packets have to be destroyed before player_unprepare() is called.</para></remarks>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <exception cref="InvalidOperationException">
+        ///     Operation failed; internal error.
+        ///     </exception>
+        /// <seealso cref="PlayerAudioExtractOption"/>
+        /// <since_tizen> 6 </since_tizen>
+        public void DecodeAudioFrame(AudioMediaFormat format, PlayerAudioExtractOption option)
```

Comment:
How to disable decode audio frame?
It only allow enabling.

Reply:
i've fixed it